misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/picking/PickLineScanScreen.jsx

```diff
@@ -109,6 +119,7 @@ const PickLineScanScreen = () => {
       resolveScannedBarcode={resolveScannedBarcode}
       onResult={onResult}
       onClose={onClose}
+      getConfirmationPromptForQty={isShowPromptWhenOverPicking ? getConfirmationPromptForQty : undefined}
```

Comment:
tbh, introducing `getConfirmationPromptForQty` it's not very cool because `getConfirmationPromptForQty` function will be created on each call of PickingLineScanScreen (a lot of times).
And because of that it will trigger a props changed to ScanHUAndGetQtyComponent.

I would suggest to better introduce 2 props like:
* confirmIfQtyAbove - number
* confirmIfQtyAboveMessageKey - message to be trl-ed and displayed.

Reply:
tbh I kinda love the flexibility a function gives us, it might be that a confirmation is required based on other criterias than qtyInput > qtyRemaning